features/que.feature

```diff
@@ -2,9 +2,9 @@ Feature: Errors are delivered to Bugsnag from Que
 
 Scenario: Que will deliver unhandled errors
   Given I run the service "que" with the command "bundle exec ruby app.rb unhandled"
-  And I run the service "que" with the command "bundle exec que ./app.rb"
-  And I wait to receive a request
-  Then the request is valid for the error reporting API version "4.0" for the "Ruby Bugsnag Notifier"
+  And I run the service "que" with the command "timeout 5 bundle exec que ./app.rb"
```

Comment:
`timeout` is necessary here because Que doesn't have an equivalent of Delayed Job's `workoff` — as this is run in the foreground now, it would be stuck infinitely